.github/workflows/autopr.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
         cd host
         ./verify-pipelines.sh ${{ github.event.inputs.majorVersion }}
         ./update-version.sh ${{ github.event.inputs.majorVersion }} ${{ github.event.inputs.targetVersion }}
+        ./validate-hostversion.sh ${{ github.event.inputs.majorVersion }}
```

Comment:
For hotfixes that are specific to a single image - this check needs to be skipped - can you add comment in this file and also, update https://github.com/Azure/azure-functions-docker/wiki/Release-Images-for-Linux-Dedicated-(GitHub-Action) to include how to skip this check?

Reply:
Added thank you